app/src/main/java/com/simply/birthdayapp/data/mappers/UpdateProfileMapper.kt

```diff
@@ -0,0 +1,12 @@
+package com.simply.birthdayapp.data.mappers
+
+import com.apollographql.apollo3.api.Optional
+import com.simply.birthdayapp.data.entities.UpdateProfileEntity
+import com.simply.birthdayapp.type.UpdateProfileInput
+
+fun UpdateProfileEntity.toUpdateProfile() =
+    UpdateProfileInput(
+        firstName = Optional.present(firstName),
+        lastName = Optional.present(lastName),
+        image = if (image != null) Optional.present(image) else Optional.absent(),
```

Comment:
You can use `Optional.presentIfNotNull()` here

Reply:
Fixed